From 7b47de3c194f239c5fea09a0e49696c9af017d51 Mon Sep 17 00:00:00 2001
From: Peter Powell <petpow@saberuk.com>
Date: Wed, 15 May 2019 09:37:30 +0100
Subject: [PATCH] Copy the silence flags when sending update notifications.

This fixes a crash bug in the silence module on some versions of GCC.
---
 src/modules/m_silence.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/modules/m_silence.cpp b/src/modules/m_silence.cpp
index 01f99ce7ee..87b70dfa46 100644
--- a/src/modules/m_silence.cpp
+++ b/src/modules/m_silence.cpp
@@ -187,7 +187,7 @@ class SilenceMessage : public ClientProtocol::Message
 		: ClientProtocol::Message("SILENCE")
 	{
 		PushParam(mask);
-		PushParamRef(flags);
+		PushParam(flags);
 	}
 };
 
